R/pkg/R/functions.R

```diff
@@ -3652,3 +3652,43 @@ setMethod("posexplode",
             jc <- callJStatic("org.apache.spark.sql.functions", "posexplode", x@jc)
             column(jc)
           })
+
+#' collect_list
+#'
+#' Creates a list of objects with duplicates.
+#'
+#' @param x Column to compute on
+#'
+#' @rdname collect_list
+#' @name collect_list
+#' @family agg_func
```

Comment:
My bad.

On a side note, do you think we should provide detailed examples for each function?